WcaOnRails/config/webpack/production.js

```diff
@@ -1,3 +1,5 @@
 const environment = require('./environment')
 
-module.exports = environment.toWebpackConfig()
+const config = environment.toWebpackConfig()
+config.devtool = 'none'
+module.exports = config
```

Comment:
It could be written as:
```js
module.exports = Object.assign({}, environment.toWebpackConfig(), {
  devtools: 'none'
});
```
A matter of preference, whatever you like most =)